src/bosh-director/lib/bosh/director/api/controllers/tasks_controller.rb

```diff
@@ -138,6 +143,27 @@ def filter_task_by_deployment_and_teams(dataset, deployment, token_scopes, limit
         end
       end
 
+      put '/', consumes: :json do
```

Comment:
@knm3000 i was just thinking more about this, should we have a new controller: /tasks_configs which provides GET and PUT with a single value `{"paused": true|false}`. that way endpoints are a bit more symmetric.

cc @tylerschultz